src/core/src/core_logic/PatchInstaller.py

```diff
@@ -557,6 +559,10 @@ def mark_installation_completed(self):
         else:
             self.status_handler.set_installation_substatus_json(status=Constants.STATUS_SUCCESS)
 
+        # If esm packages are found, set the status as warning. This will show up in portal along with the error message we already set.
+        if self.esm_packages_found:
```

Comment:
note: may want this to be esm_packages_found_without_attach  to make this very explicit on intent.

Reply:
Done